api/pkg/apps/cms/types.go

```diff
@@ -53,3 +54,17 @@ type Comment struct {
 type CommentList struct {
 	Items []*Comment `json:"items"`
 }
+
+// Case templates
+// https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+type CaseTemplate struct {
+	// FormElements is an ordered list of the elements found in the form
+	FormElements []CaseTemplateFormElement `json:"formElements" bson:"formElements"`
+}
+
+type CaseTemplateFormElement struct {
+	Type       string                 `json:"type" bson:"type"`
+	Attributes map[string]interface{} `json:"attributes" bson:"attributes"`
```

Comment:
I think `Attributes` and `Validation` should be structs as well...

Reply:
The thing is, attributes and validation fields (and field types) will vary depending on the form element. Should we have structs for each element?

Reply:
@nilueps I don't think so, we can put all the available attributes in the same struct, and have validation rules to make sure the payload is in the right format